shared/actions/wallets.js

```diff
@@ -1,15 +1,72 @@
 // @flow
-// import * as Constants from '../constants/wallets'
-// import * as Types from '../constants/types/wallets'
-import * as WalletsGen from './wallets-gen'
+import * as Constants from '../constants/wallets'
+import * as Types from '../constants/types/wallets'
+import * as RPCTypes from '../constants/types/rpc-stellar-gen'
 import * as Saga from '../util/saga'
+import * as WalletsGen from './wallets-gen'
+import type {TypedState} from '../util/container'
+
+const loadAccounts = (action: WalletsGen.LoadAccountsPayload) =>
+  Saga.call(RPCTypes.localGetWalletAccountsLocalRpcPromise, {})
+
+const loadAccountsSuccess = (res: ?Array<Types.Account>) =>
+  Saga.put(
+    WalletsGen.createAccountsReceived({
+      accounts: (res || []).map(account => Constants.accountResultToAccount(account)),
+    })
+  )
+
+const loadAssets = (action: WalletsGen.LoadAssetsPayload) => {
+  const {accountID} = action.payload
```

Comment:
if we don't have these lines to extract the one param they could all just return the non {} version, aka

```
const loadAssets = (action: WalletsGen.LoadAssetsPayload) = 
  Saga.call(RPCTypes.localGetAccountAssetsLocalRpcPromise, {accountID: action.payload.accountID})
```

but just a style thing